azkaban-web-server/src/main/java/cloudflow/readme.md

```diff
@@ -105,3 +105,246 @@ curl -X POST 'http://localhost:8081/executions' \
 }'
 
 Response body is empty
+
+
+** Get all the flows for a project
+curl -X GET http://localhost:8081/flows
+[ {
+  "id" : "1",
+  "name" : "flow1_proj5",
+  "flowVersionCount" : 2,
+  "projectId" : "5",
+  "projectVersion" : "2",
+  "admins" : null,
+  "createdByUser" : "gsalia",
+  "createdOn" : 1582577888795,
+  "modifiedOn" : 1582577888800,
+  "lastVersion" : {
```

Comment:
Consider `latestVersion` instead ?

Reply:
Changed to latestVersion, that's what I had before but renamed it based on a prior discussion.